immersive-web-wg-charter.html

```diff
@@ -388,6 +388,13 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             <dd>
               The Immersive Web Community Group will provide the WebXR Device API seed specification to begin the standards process.  In addition, the Immersive Web Working Group plans to partner closely with the IWCG to incubate new features - in particular,  incubation of features that are out of current scope for the working group will happen in the Community Group, and then be followed by future WG rechartering to include them in scope.
             </dd>
+            <dt>
+              <a href="https://www.w3.org/community/idiw/">Inclusive Design for Immersive Web Community
+              Group</a>
+            </dt>
+            <dd>
+              The Inclusive Design for Immersive Web Community Group tracks and promotes progress on accessibility issues identified across the many relevant W3C and in particular with the work done in the Immersive Web Working Group.
```

Comment:
"across the many relevant W3C and" => "across the many relevant W3C standards and"

Reply:
Fixed in master.

Reply:
note. updated 'proposed-immersive-web-wg-charter.html' also.